reconcile/dynatrace_token_provider/ocm.py

```diff
@@ -50,7 +51,7 @@ class Cluster(BaseModel):
     @staticmethod
     def from_cluster_details(cluster: ClusterDetails) -> Cluster:
         dt_tenant = cluster.labels.get_label_value(DTP_TENANT_LABEL)
-        token_spec_name = cluster.labels.get_label_value(DTP_LABEL)
+        token_spec_name = cluster.labels.get_label_value(DTP_SPEC_LABEL)
```

Comment:
Is `DTP_LABEL` used anywhere else?

Reply:
Good catch! Yeah it should not be needed any longer.